test/e2e/node/mount_propagation.go

```diff
@@ -180,15 +181,28 @@ var _ = SIGDescribe("Mount propagation", func() {
 				}
 			}
 		}
-		// Check that the mounts are/are not propagated to the host.
-		// Host can see mount from master
-		cmd = fmt.Sprintf("test `cat %q/master/file` = master", hostDir)
-		err = hostExec.IssueCommand(cmd, node)
-		framework.ExpectNoError(err, "host should see mount from master")
 
-		// Host can't see mount from slave
-		cmd = fmt.Sprintf("test ! -e %q/slave/file", hostDir)
-		err = hostExec.IssueCommand(cmd, node)
-		framework.ExpectNoError(err, "host shouldn't see mount from slave")
+		// Find the kubelet PID to ensure we're working with the kubelet's mount namespace
+		cmd = "pidof /usr/bin/kubelet"
```

Comment:
per discussion in sig, we will want to use an alternate mechanism to discover the kubelet pid.  we may have a helper elsewhere, will have to look.

Reply:
Agreed.

Reply:
I looked elsewhere in the code, and it looks like the convention in other e2e tests here is to just use `pidof execname` without an explicit path.